README.md

```diff
@@ -1,51 +1,17 @@
 # Planetary Computer Examples
 
-This repository contains Jupyter Notebooks that serve as examples of using the [Microsoft Planetary Computer](https://planetarycompture.microsoft.com) data, APIs and Hub.
+<img src="https://planetarycomputer.microsoft.com/images/planet-sunrise-wide@1.jpg" alt="Planetary Computer" width="50%"/>
 
-## Getting Started
+Welcome to the [Planetary Computer Hub](http://planetarycomputer.microsoft.com/compute), a development environment that makes our data and APIs accessible through familiar, open-source tools, and allows users to easily scale their analyses.
```

Comment:
This language kind of implies that people will be on the hub when viewing this document, which isn't strictly true. But hopefully it's OK. I added a link to the hub, in case people are viewing this on GitHub.